udfs/tests/dataform_testing_framework/deploy_and_run_tests.sh

```diff
@@ -219,6 +219,26 @@ main() {
   # for all other Dataform project directories.
   dataform install >/dev/null 2>&1
 
+  # Deploy UDFs to bigquery-public-data project
+  public_project_id="bigquery-public-data"
+  public_dataset_id="persistent_udfs"
+  if [[ -n "${_PUBLIC_DATA}"]]; then
```

Comment:
everything that follows outside of this if statement should be part of an else statement. If _PUBLIC_DATA is set then we're only deploying to public data and not doing any of the other steps that come after this if statement.

Reply:
alternatively, we can just add an `exit 0` after the last statement in the if block

Reply:
resolved by adding if/then logic